### ISSUE TITLE
question about the learningRate

### ISSUE THREAD

**Issue by yao-ying** [bazilas/matconvnet-ResNet#4]
Hi all.I have a question about the
opts.learningRate = [0.01*ones(1,3) 0.1*ones(1,80) 0.01*ones(1,10) 0.001*ones(1,20)] ;
As I known,if the learning rate is small,it is easy to overfit.So I think the 
opts.learningRate = [0.01*ones(1,3) 0.1*ones(1,80) 0.01*ones(1,20) 0.001*ones(1,10)] ; is better.
